### PR TITLE
Add one scenario for multi steps

### PR DESCRIPTION
**Pull request by quynhn**

### CODE REVIEW

**Comment by chibenwa** [comment]
Attach gatling screenshots to the review PLZ

**Comment by chibenwa** [comment]
Also add getMailboxes in the loop plz

**Comment by chibenwa** [comment]
Well, otherwise I think it's cool

We can clearly see the impact of SendMessages on garbage collection that takes all the perf down. I believe, even if I don't know how to reduce this, that this should be the next big task of the team : making SendMessages faster... Maybe embeddedActiveMQ improvements...

**Comment by quynhn** [comment]
```
Also add getMailboxes in the loop plz
```
Done with the last commit

**Review by chibenwa — APPROVED** [review]
Please send again the next run results ;-) (and send it again)

**Comment by quynhn** [comment]
```
Attach gatling screenshots to the review PLZ
```
Can I wait for 1000 users running in 3 hours? I will attach it right after that

File: src/test/scala-2.11/org/apache/james/gatling/jmap/scenari/JmapAllScenario.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  val scn = scenario("JmapAllScenarios")
+    .exec(CommonSteps.provisionSystemMailboxes(users))
+    .during(ScenarioDuration) {
+      exec(JmapMessages.sendMessagesRandomlyWithRetryAuthentication(users))
+      .pause(1 second, 5 seconds)
+      .exec(JmapMessages.listMessagesWithRetryAuthentication())
+      .pause(1 second, 5 seconds)
+      .exec(JmapMessages.getMessagesWithRetryAuthentication())
+      .pause(1 second, 5 seconds)
```
Comment (rouazana) [inline src/test/scala-2.11/org/apache/james/gatling/jmap/scenari/JmapAllScenario.scala]: I guess that's not the pausing that is issuing Inbox: if you look at your browser network debug tag, you'll see that they often send requests 3 at a time (getMailboxes + getMessageList + getMessages). Idem when sending a message where getMailboxes is called just after setMessages.

**Review by mbaechler — APPROVED** [review]

**Review by rouazana — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Comment by quynhn** [comment]
I shared to you all the detail result which run on 3 hours via linshare.

**Pull request merged**